File: JavaScript/multiples-of-3-or-5.js
function solution(number) {
    let total = [];
    let multiplier = 1;
    while (3 * multiplier < number) {
        total.push(3 * multiplier);
        multiplier++;
    }
    multiplier = 1;
    while (5 * multiplier < number) {
        total.push(5 * multiplier)
        multiplier++;
    }
    newSet = new Set(total);
    let sum = 0;
    for (let item of newSet) {
        sum += item
    }
    return sum
}

File: Python/multiples-of-3-or-5.py
def solution(number):

    total = []
    multiplier = 1
    while 3 * multiplier < number:
        total.append(3 * multiplier)
        multiplier += 1

    multiplier = 1
    while 5 * multiplier < number:
        total.append(5 * multiplier)
        multiplier += 1

    return sum(set(total))
  
